sdk/eventhubs/azure-messaging-eventhubs-stress/stress-test-resources.bicep

```diff
@@ -0,0 +1,109 @@
+@description('The base resource name.')
+param baseName string = resourceGroup().name
+
+@description('The location of the resources. By default, this is the same as the resource group.')
+param location string = resourceGroup().location
+
+var subBaseName = substring(baseName, 0, min(length(baseName), 10))
```

Comment:
It may be more defensive to call `guid()` here as well, in case the truncated names cause resource name overlaps when only the suffixes differ.

Reply:
I find resource group id is formed by subscription id and resource group name. The unique identifier is resource group name. I think testers need to prevent using the same prefix when they use scenarios values (current I remove `values.yaml` since the content of `job.yaml` are different in our test cases). But I can make a note for this potential issue.

Reply:
Sorry, I misunderstand before. You mean we can use guid() to generate a guid. I'll add this into name.